pkg/keepalive/options.go

```diff
@@ -68,4 +78,9 @@ func (o *Options) AttachCobraFlags(cmd *cobra.Command) {
 			"and if no activity is seen even after that the connection is closed.")
 	cmd.PersistentFlags().DurationVar(&o.MaxServerConnectionAge, "keepaliveMaxServerConnectionAge",
 		o.MaxServerConnectionAge, "Maximum duration a connection will be kept open on the server before a graceful close.")
+	cmd.PersistentFlags().IntVar(&o.WriteBufferSize, "writeBufferSize", o.WriteBufferSize,
+		"WriteBufferSize determines how much data can be batched before doing a write on the wire.")
+	cmd.PersistentFlags().IntVar(&o.ReadBufferSize, "readBufferSize", o.ReadBufferSize,
+		"ReadBufferSize lets you set the size of read buffer, this determines how much data can be read at "+
```

Comment:
can you please provide guidance on how this should be tuned by the user in the description here? I think this desp is going to our reference doc.

Reply:
Definitely very few users may need adjust these values. The desc should be more clearly.

Reply:
I will add some suggested usages for these two parameters here, after I run some relevant grpc benchmark.

Reply:
[Here](https://github.com/yangdihangN/servicemesh-doc/blob/master/Benchmark-for-Grpc-WriteRead-Size.md) is some benchmark for WB/RB. 
I think the scene similar to istio is that there are more Connections, but each Connection has fewer concurrent calls (envoy doesn't seem to harass pilot often). From the results, the size of ReadBuffer has almost no effect on the qps of Server (pilot). Can we reduce the default value of ReadBuffer and do related verification? In addition, I added some suggestions at the end of the article, which may be used in our references.